pramen/core/src/test/scala/za/co/absa/pramen/core/tests/utils/JsonUtilsSuite.scala

```diff
@@ -30,7 +33,7 @@ class JsonUtilsSuite extends AnyWordSpec {
       |  "longValue" : 10000000000,
       |  "dateValue" : "2020-08-10",
       |  "listStr" : [ "Str1", "Str2" ]
-      |}""".stripMargin
+      |}""".replace(EOL, "\n").stripMargin
```

Comment:
So, the line separators in the string literal depend on the configuration of your IDE. If you have your IDE configured to CRLF line endings, the string will have CRLF line endings - same with LF etc. 

The string which is returned by the `JsonUtils.prettifyJson()` call has line endings which depend on your system (CRLF for Windows, LF for Unix). JVM probably chooses the correct line separator.

So in case you have your IDE configured differently from your system line endings (like me 😄), the test will fail. The string literal will have, for example, LF line endings while the result from the function call will have CLRF. This workaround solves this but is not very pretty. 

If someone will have CR line endings  setup in the IDE, but will run it on Windows (don't know if such people exist 😄),  then this test will fail.

Reply:
It is more complicated than I thought 😆 

I'm wondering if we can just remove all CR from CRLFs? Would it work with your settings?
```
.replace("\r", "")
```
Then CRLF will be converted to LF, and LF will be converted to LF (supposedly).

But in any case, thanks for the fix!

Reply:
oo, nice idea